docs/src/content/docs/tutorial/abstraction.md

```diff
@@ -81,39 +84,20 @@ Let's start cleaning this up by separating logical pieces into distinct files:
 }
 ```
 
-The file should contain just the same that was located under the `grafana` key
-on the root object before. Do the same for `/environments/default/prometheus.jsonnet` as well.
+The file should contain an object with just the same function that was defined under the `grafana` in `/environments/default/main.jsonnet`, but called `new` instead of `grafana`.
```

Comment:
```suggestion
The file should contain an object with the same function that was defined under the `grafana` in `/environments/default/main.jsonnet`, but called `new` instead of `grafana`.
```